edm4dr/CMakeLists.txt

```diff
@@ -2,7 +2,16 @@
 # installed podio
 find_package(EDM4HEP REQUIRED)
 
-PODIO_GENERATE_DATAMODEL(edm4hep ../edm4dr.yaml headers sources IO_BACKEND_HANDLERS ${PODIO_IO_HANDLERS})
+# Use the EDM4HEP_DATA_DIR to derive extended data model
+message(STATUS "${EDM4HEP_DATA_DIR}")
+if ("${EDM4HEP_DATA_DIR}" STREQUAL "")
+  message(WARNING "Unable to extend data model since no installed yaml file found.")
```

Comment:
In case of edm4dr this should probably be a hard failure once the yaml file has changed, right?

Reply:
yes, good point!